src/task1/Animal.java

```diff
@@ -0,0 +1,61 @@
+package task1;
+
+public class Animal {
+    private String breed;
+    private int years;
+    private double weight;
+
+    public Animal(String breed, int years) {
+        this.breed = breed;
+        this.years = years;
+
+    }
+
+    public Animal(String breed, int years, double weight) {
+        this.breed = breed;
+        this.years = years;
+        this.weight = weight;
+    }
+
+    public String getBreed() {
+        return breed;
+    }
+
+    public int getYears() {
+        return years;
+    }
+
+    public double getWeight() {
+        return weight;
+    }
+
+    public void setBreed(String breed) {
+        this.breed = breed;
+    }
+
+    public void setYears(int years) {
+        this.years = years;
+    }
+
+    public void setWeight(double weight) {
+        this.weight = weight;
+    }
+
+    public void animalAging(int years) {
+        this.years += years;
+    }
+
+    public void animalFatteing(double weight) {
+        this.weight += weight;
+    }
+
+    public void animalLosingWeight(double weight) {
```

Comment:
nitpick: methods should generally be verbs and not nouns. In your case `ageAnimal()`, `fattenAnimal()` something like this. 
No big deal in this case, really.